services/loadbalancer/wait/wait.go

```diff
@@ -46,6 +47,15 @@ func CreateLoadBalancerWaitHandler(ctx context.Context, a APIClientInterface, pr
 		if s == nil || s.Name == nil || *s.Name != instanceName || s.Status == nil {
 			return false, nil, nil
 		}
+
+		if s.Errors != nil && len(*s.Errors) != 0 {
```

Comment:
Could it happen that the instance has STATUS_READY and errors in the `Errors` field? In that case the waiter will fail.

We should clarify with the team if that can happen and what should we do in that case.

Reply:
Discussed it with colleague from the Load Balancer team. They think there is also an issue on their side and will investigate why errors might be set and not be reflected on the state. Nevertheless, he said we could use this errors list to determine if there is an error or not